src/auth/crud.py

```diff
@@ -41,12 +51,19 @@ async def remove_user_session(db_session: AsyncSession, session_id: str) -> dict
     await db_session.delete(user_session.first())  # TODO: what to do with this result?
 
 
-async def check_session_validity(db_session: AsyncSession, session_id: str) -> dict:
+async def check_user_session(db_session: AsyncSession, session_id: str) -> dict:
     query = sqlalchemy.select(models.UserSession).where(models.UserSession.session_id == session_id)
     existing_user_session = (await db_session.scalars(query)).first()
 
     if existing_user_session:
-        return {"is_valid": True, "computing_id": existing_user_session.computing_id}
+        query = sqlalchemy.select(models.User).where(models.User.computing_id == existing_user_session.computing_id)
```

Comment:
instead of explicitly doing another select, we can replace this with an sqlachemy relationship access. Let's leave it this way for now, but include a TODO.

Reply:
ooo sure, I'll drop a TODO